tools/clang/test/HLSLFileCheck/hlsl/workgraph/test_EmitDxilFunctionProps_MDVals.hlsl

```diff
@@ -0,0 +1,29 @@
+// RUN: %dxc -fcgl -T lib_6_8 %s | FileCheck -check-prefix=CHECK_FCGL %s
```

Comment:
Isn't this just a partial/equivalent copy of the `called_function_arg_nodeoutput.hlsl` test?  This doesn't change verification to check the metadata values, which the test name implies either.  Is there any added value of this new test over just running the other test?

Reply:
This test has no added value when compared to the other test. It was originally intended to just check for successful compilation, since this specific compilation path would have caused an out-of-bounds assignment in the older code. 
One improvement I could make is to remove the checks and add a CHECK: !36 to prove that there are more than 35 metadata values in the MDVals data structure and no out of bounds behavior is happening.